app/components/multiple_checked_select_component/multiple_checked_select_component.html.haml

```diff
@@ -0,0 +1,16 @@
+.ofn-drop-down.ofn-drop-down-v2{ data: { controller: "multiple-checked-select", "multiple-checked-select-input-name-value": @name } }
+  %div.ofn-drop-down-label{ "data-multiple-checked-select-target": "button" }
+    %span{class: "label"}= t('admin.columns')
+    %span{ class: "icon-caret-down", "data-multiple-checked-select-target": "caret" }
+  %div.menu{ class: "hidden", "data-multiple-checked-select-target": "options" }
+    %div.filter
+      %input{ type: "text", "data-multiple-checked-select-target": "filter", placeholder: @filter_placeholder }
+    %hr
+    %div.menu_items
+      - @options.each do |option|
+        - classes = @selected.include?(option[1]) ? "selected" : ""
+        %div.menu_item{ class: classes, "data-multiple-checked-select-target": "option", "data-value": option[1], "data-label": option[0] }
+          %span.check
+          %span.name{id: option[1]}
```

Comment:
This id may not be unique. Do we need to add a prefix?

Reply:
Actually this one is useless. Removing.